src/mlpack/methods/ann/layer/padding_impl.hpp

```diff
@@ -34,7 +34,8 @@ Padding<InputDataType, OutputDataType>::Padding(
     nRows(0),
     nCols(0),
     inputHeight(inputWidth),
-    inputWidth(inputHeight)
+    inputWidth(inputHeight),
+    inSize(0)
```

Comment:
I would open another PR just for this one, and remove it from here, this will help us better keep tracking what this PR is doing on the CI fix, since this is not related directly to the CI

Reply:
It's just warning removal actually. You can do whatever you want to do.